sideloader/sideloader.py

```diff
@@ -238,13 +240,8 @@ def copy_build(self):
         self.workspace.make_install_dir()
 
         for directory in os.listdir(self.workspace.get_build_path()):
-            try:
-                shutil.copytree(self.workspace.get_build_path(directory),
-                                self.workspace.get_install_path(directory))
-            except Exception as e:
-                log('ERROR: Could not copy %s to package: %s %s' % (
-                    directory, type(e), e))
-                self.fail_build('Error copying files to package')
+            shutil.copytree(self.workspace.get_build_path(directory),
+                            self.workspace.get_install_path(directory))
```

Comment:
I hadn't implemented `fail_build()` or any of the infrastructure for the `allow_broken_build` flag. Will revisit in separate PR.